WooCommerce/Classes/ViewRelated/Dashboard/DashboardViewModel.swift

```diff
@@ -258,6 +263,19 @@ final class DashboardViewModel {
 
     }
 
+    @MainActor
+    private func loadLocalAnnouncements() async {
```

Comment:
❓ Can we show more than one local announcement? If we do, should we have an array for `localAnnouncementViewModel`? Otherwise, the method name should be singular to avoid ambiguity.

Reply:
No, just one local announcement at a time. I think the plural "announcements" makes it sound like there could be multiple announcements to be displayed, I renamed it and added a comment on the function in https://github.com/woocommerce/woocommerce-ios/pull/10118/commits/8762aa6089ae4c61b452f368aec6c588d606d033.